.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
+/.idea
```

Comment:
Please add this to your `$HOME/.gitignore`; smoltcp should not collect files of every IDE that exists.